update-bunsen-pepperflash

```diff
@@ -126,7 +126,7 @@ check_versions(){
         return 0
     fi
 
-    if dpkg --compare-versions "$newver" gt "$instver"
+    if [[ "$newver" > "$instver" ]]
```

Comment:
Works only if it is guaranteed that the versions are of the form x.y.z. Probably works though.